### PR TITLE
Update workload page

### PR DESCRIPTION
**Pull request by inyee786**
Get the number of replica and controller from mayaapi-server
Get the openebs version
Get the k8s version
Modify the UI of application status in workload
Make workload right side scrollable

Signed-off-by: inyee786 <intakhab.ali@mayadata.io>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: src/app/workloads/workloads.component.html [hunk 1]
```diff
@@ -23,25 +23,24 @@
...
             </div>
           </div>
         </div>
         <div class="d-flex flex-column my-auto pr-4">
           <div class="d-flex flex-column ">
-            <div class="d-flex align-items-center mb-3 ml-4">
+            <div class="d-flex align-items-end mb-3 ml-4">
               <img src="/assets/images/company-logos/openebs-logo-mule.svg" alt="" height="30">
-              <p class="mb-0 font-size-12 text-dark"> &nbsp;&nbsp; {{openebsengine}} -0.8.1</p>
+              <p class="mb-0 font-size-12 text-dark"> {{openebsengine}} - {{openebsversion}}</p>
```
Comment (bhaskarhc) [inline src/app/workloads/workloads.component.html]: Remove `openebsEngine`, it is not required!

File: src/app/workloads/workloads.component.ts [hunk 2]
```diff
@@ -67,8 +69,6 @@ export class WorkloadsComponent implements OnInit, OnDestroy {
   public pvctemp;
```
Comment (bhaskarhc) [inline src/app/workloads/workloads.component.ts]: change variable to  `pvcTemp`

File: src/app/workloads/workloads.component.ts [hunk 3]
```diff
@@ -67,8 +69,6 @@ export class WorkloadsComponent implements OnInit, OnDestroy {
...
   public pvcarray;
```
Comment (bhaskarhc) [inline src/app/workloads/workloads.component.ts]: here also `pvcArray`

File: src/app/workloads/workloads.component.ts [hunk 4]
```diff
@@ -213,30 +213,14 @@ export class WorkloadsComponent implements OnInit, OnDestroy {
...
           this.pvcarray = this.pvctemp.pvc;
-          // this.workloadImage = this.statefullSets[0].dockerImage;
-          // this.dockerImage = this.jivaContrllers[0].openebsjivaversion;
-          this.openebsversion = this.jivaContrllers[0].openebsjivaversion.split(
-            ":"
-          )[1];
+          this.openebsVersion = this.jivaContrllers[0].openebsVersion
           this.namespace = this.statefullSets[0].namespace;
           this.overallStatus = res.status;
           this.numberstatefullSets = this.statefullSets.length;
```
Comment (bhaskarhc) [inline src/app/workloads/workloads.component.ts]: numberStatefullSets

File: src/app/workloads/workloads.component.ts [hunk 5]
```diff
@@ -282,12 +267,22 @@ export class WorkloadsComponent implements OnInit, OnDestroy {
       .subscribe(res => {
         this.litmuspod = res.statefulSet;
         this.mayapvc = res.pvcs.pvc
```
Comment (bhaskarhc) [inline src/app/workloads/workloads.component.ts]: mayaPvc

File: src/app/services/savereaddelete.service.ts [hunk 6]
```diff
@@ -63,10 +57,10 @@ export class PersonService {
   }
   getpersonDetails() {
```
Comment (bhaskarhc) [inline src/app/services/savereaddelete.service.ts]: getPersonDetails()

File: src/app/services/kubernetes.service.ts [hunk 7]
```diff
@@ -10,21 +10,21 @@ import { concatMap, mergeMap } from "rxjs/operators";
...
 
     setApiUrl(apiUrl: string) {
-        this.apiurl = apiUrl;
+        this.apiUrl = apiUrl;
     }
     getApiUrl() {
-        return this.apiurl;
+        return this.apiUrl;
     }
     getVolumeDetails(workloadname: string, openebsengine: string, pvcDetails: mayapvc) {
```
Comment (bhaskarhc) [inline src/app/services/kubernetes.service.ts]: use : `workloadName` , `openebsEngine`

**Comment by chandankumar4** [comment]
Closing this PR because we have to move this repo to mayadata and also this PR need some changes you  can raise it again!!

**Pull request merged**